src/main/java/com/google/sps/utils/SoyRendererUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.utils;
+
+import java.io.File;
+import com.google.common.collect.ImmutableMap;
+import com.google.template.soy.SoyFileSet;
+import com.google.template.soy.tofu.SoyTofu;
+
+public final class SoyRendererUtils {
+    private static final ClassLoader CLASS_LOADER = SoyRendererUtils.class.getClassLoader();
+
+    private SoyRendererUtils() {
```

Comment:
What is this function for?

Reply:
doesn't seem to do anything, will remove!